BMM.Core/Implementations/Feedback/EmailContacter.cs

```diff
@@ -32,7 +32,7 @@ public async Task Contact()
             sb.AppendLine("Device version: " + _deviceInfo.VersionString);
             sb.AppendLine("Full name: " + user.FullName);
             sb.AppendLine("Username: " + user.Username);
-            sb.AppendLine("PersonId: " + user.PersonId);
+            sb.AppendLine("AnalyticsIdentifier: " + user.AnalyticsIdentifier);
```

Comment:
I'm not sure if we should always send it or if the user has to voluntarily reveal that information